Containerfile

```diff
@@ -22,21 +22,23 @@ FROM registry.access.redhat.com/ubi8/ubi:latest as builder
 RUN dnf -y --setopt=tsflags=nodocs install \
       http://mirror.centos.org/centos/8-stream/BaseOS/x86_64/os/Packages/centos-gpg-keys-8-4.el8.noarch.rpm \
       http://mirror.centos.org/centos/8-stream/BaseOS/x86_64/os/Packages/centos-stream-repos-8-4.el8.noarch.rpm \
- && dnf -y --setopt=tsflags=nodocs install epel-release \
  && dnf config-manager --set-enabled powertools \
  && dnf clean all
 
 RUN dnf -y --setopt=tsflags=nodocs install \
     gcc gcc-c++ make cmake \
     cyrus-sasl-devel openssl-devel libuuid-devel \
     python3-devel swig \
-    libwebsockets-devel libnghttp2-devel \
+    libnghttp2-devel \
     wget patch findutils git valgrind \
  && dnf clean all -y
+
 WORKDIR /build
 COPY . .
 ENV PROTON_VERSION=0.37.0
 ENV PROTON_SOURCE_URL=${PROTON_SOURCE_URL:-http://archive.apache.org/dist/qpid/proton/${PROTON_VERSION}/qpid-proton-${PROTON_VERSION}.tar.gz}
+ENV LWS_VERSION=v3.2.3
```

Comment:
somewhat OFFTOPIC: I wonder about the version choice. Epel had `4.0.1-2.el8` Why not use latest here? That would be `v4.3.1`, https://github.com/warmcat/libwebsockets/releases/tag/v4.3.1

Reply:
Table of all versions in the opensource RH world, https://src.fedoraproject.org/rpms/libwebsockets

Reply:
@ganeshmurthy wdyt, also taking https://github.com/skupperproject/skupper-router/issues/48 into consideration. We don't have to resolve this on the PR right away, of course. Just something to keep in mind and eventually take care of. 

Depending also on what @mcressman thinks, of course.

Reply:
I am all for using the latest version of LWS